osu.Game.Tests/Visual/Editing/TestScenePreviewTime.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Testing;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Osu;
+using osu.Game.Screens.Edit.Components.Timelines.Summary.Parts;
+
+namespace osu.Game.Tests.Visual.Editing
+{
+    public partial class TestScenePreviewTime : EditorTestScene
+    {
+        protected override Ruleset CreateEditorRuleset() => new OsuRuleset();
+
+        [Test]
+        public void TestSceneSetPreviewTimingPoint()
+        {
+            AddStep("seek to 1000", () => EditorClock.Seek(1000));
+            AddAssert("time is 1000", () => EditorClock.CurrentTime == 1000);
+            AddStep("set current time as preview point", () => Editor.SetCurrentTimeAsPreview());
+            AddAssert("preview time is 1000", () => EditorBeatmap.PreviewTime.Value == 1000);
+        }
+
+        [Test]
+        public void TestScenePreviewTimeline()
+        {
+            AddStep("set preview time to -1", () => EditorBeatmap.PreviewTime.Value = -1);
+            AddAssert("preview time line should not show", () => Editor.ChildrenOfType<PreviewTimePart>().Single().Alpha == 0);
```

Comment:
why shouldn't it show? if `PreviewTime` is -1 then it's automatically 40% of the track length. surely you can show the line there

Reply:
This should be the expected situation.
The preview point should not be displayed on the map without preview point (also stable)

If it is really shown in the 40% position, it may be misleading (let mapper think its already set)
if we must do this, I think it should be a different color, but that would get weird

Reply:
I think of these options, not showing at all is probably best.